st/clitests/auth_spec.py

```diff
@@ -1812,6 +2045,82 @@ def auth_health_check_tests():
     result = AuthTest('Auth server health check').get_auth_health(health_check_uri).\
     execute_test().command_is_successful().command_response_should_have("200 OK")
 
+# Validate maxAccount and maxUser limit values from authserver.properties file
+def test_max_account_and_user_limit_value_of_auth_config():
+    print("Updating autherver.properties (/opt/seagate/cortx/auth/resources/authserver.properties) file with test values..")
```

Comment:
can we take install dir path from env or config if possible instead of hardcoding

Reply:
with current code changes, this is not possible.. because we need to add those config/properties file to environment first and then we can use this with some alias in case of jenkins-build process  .
should we take this as part of I&P story ?
cc @nileshgovande